assets/src/edit-story/components/canvas/layout.js

```diff
@@ -170,9 +207,28 @@ function useLayoutParamsCssVars() {
   return {
     '--page-width-px': `${pageSize.width}px`,
     '--page-height-px': `${pageSize.height}px`,
+    '--page-preview-width-px': `${pageSize.width}px`,
+    '--page-preview-height-px': `${pageSize.width * PREVIEW_RATIO}px`,
   };
 }
 
+function PageArea({ children }) {
+  return (
+    <PageAreaContainer>
+      <PageAreaContent>{children}</PageAreaContent>
+      <PageAreaSafeZoneTop />
+      <PageAreaSafeZoneBottom />
+    </PageAreaContainer>
+  );
+}
+
+PageArea.propTypes = {
+  children: PropTypes.oneOfType([
```

Comment:
Unrelated, but... Someone keeps telling me that we should just do `children: PropTypes.node` in all such cases and PropTypes "gets it".

Reply:
Good point, we should do that throughout the app. Thanks!